src/main/java/io/confluent/connect/jdbc/source/JdbcSourceConnectorConfig.java

```diff
@@ -165,6 +165,14 @@
   public static final String TIMESTAMP_COLUMN_NAME_DEFAULT = "";
   private static final String TIMESTAMP_COLUMN_NAME_DISPLAY = "Timestamp Column Name";
 
+  public static final String TIMESTAMP_INITIAL_CONFIG = "timestamp.initial";
+  public static final long TIMESTAMP_INITIAL_DEFAULT = 0;
```

Comment:
Should the default be `null`? That seems to be better than `0` if we want this to be an optional property.